pt-BR/active_job_basics.md

```diff
@@ -419,11 +419,10 @@ and add this serializer to the list:
 Rails.application.config.active_job.custom_serializers << MoneySerializer
 ```
 
-Exceptions
+Exceções
 ----------
 
-Active Job provides a way to catch exceptions raised during the execution of the
-job:
+*Jobs* ativos fornecem uma maneira de capturar exceções criadas durante a execução do *job*.
```

Comment:
```suggestion
*Active Job* fornecem uma maneira de capturar exceções criadas durante a execução do *job*.
```

Active Job é o nome de uma parte interna do Rails. Acho que podemos só manter em itálico o que acha?

Reply:
Realmente ajuda na compreensão, pois o termo estará mais destacado.